api/src/Service/Auth/AuthService.php

```diff
@@ -90,12 +47,9 @@ public function isSecretValid(Request $request)
     }
 
     /**
-     * @param string $email
-     * @param string $pass
-     *
      * @return User|bool|null or null if the user it not found or password is wrong
      */
-    public function getUserByEmailAndPassword($email, $pass)
+    public function getUserByEmailAndPassword($email, $pass): User|bool|null
```

Comment:
We've removed the DocBlock `@param` for the types but then we don't type hit the actual parameters in the function should we do this?

Reply:
I've reverted this change and will leave fixing these up to the PHPStan tickets. Cant face having to debug any code this may break after type hinting 🙄